examples/DynaFlow/description/DynaFlowExamples.tex

```diff
@@ -211,6 +211,91 @@ \section{Load restoration (LoadRestoration)}
 
 When the restoration is not complete, the active power of the load is below the maximum transferrable power. As a consequence, the simulation succeeds.
 
+\chapter{WSCC9}
+
+The WSCC 9-bus system represents a simple approximation of the Western System Coordinating Council (WSCC) to an equivalent system with nine buses and three generators.
+It was notably published in Anderson and Fouad’s book ‘Power System Control and Stability’ for the first time in 1980.
+
+\section{Test case description}
+
+The WSCC 9-bus system has 9 buses, 3 generators, 3 transformers, 6 lines and 3 loads.  \\
+There are four voltage levels in the test case: 230 kV, 18 kV, 16.5 kV and 13.8 kV. The inner
+part is the 230 kV part. The generators are at 18 kV, 16.5 kV and 13.8 kV.
+
+\begin{figure}[H]
+  \includegraphics[width=\textwidth]{WSCC9Bus.jpg}
```

Comment:
The figure is quite blurred

Reply:
Yes, I think it comes from the hook applied on the commits. I should delete the file and commit it again while forcing not to apply the blank lines corrections and so on.

Reply:
Solved